proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,205 @@
+# MDM Base Setup
+
+Note: these can all be namespaced behind something like `apple-mdm` if needed. But I would like to avoid creating an mdm vertical vs. an osquery vertical to create an integrated product.
```

Comment:
A prefix will be needed, e.g. `fleetctl setup` is currently used for Fleet account setup:
```
fleetctl setup
NAME:
   fleetctl setup - Set up a Fleet instance

USAGE:
   fleetctl setup [options]

OPTIONS:
   --email value     Email of the admin user to create (required) [$EMAIL]
   --name value      Name or nickname of the admin user to create (required) [$NAME]
[...]
```

Reply:
Can we have fleet setup and fleet setup apns coexist?

The basic idea here is that having all MDM stuff behind some namespace such as apple-mdm creates a fragmented product. Our value is to unify IT and security. The command fleetctl setup is for setting up my fleet, which includes Apple MDM.

Reply:
> Can we have fleet setup and fleet setup apns coexist?

Yes. I believe this is possible, basically add a subcommand to `setup`:
```
# For setting up the main setup account:
fleetctl setup --email foo@example.com --name Foo --password p4ssw0rd.123 --org-name "Foo Inc."
# For setting up APNS:
fleetctl setup apns ...
# etc.
```

> The basic idea here is that having all MDM stuff behind some namespace such as apple-mdm creates a fragmented product. Our value is to unify IT and security. The command fleetctl setup is for setting up my fleet, which includes Apple MDM.

Good point.